test/Lib/rlexe.xml

```diff
@@ -66,7 +66,7 @@ Re-enable after 190575 is fixed
    <default>
       <files>profiledataobject.js</files>
       <baseline>profiledataobject.baseline</baseline>
-      <tags>exclude_default,exclude_dynapogo,exclude_amd64,exclude_arm,require_backend</tags>
```

Comment:
These are all under a commented out section referencing bug 190575, but that isn't a relevant bug in the OS repo. Any ideas?

Reply:
The OS bug tracker is only the most recent of many bug trackers. Possibly TFS although I don't know how to access it. @boingoing ?

Reply:
This is a Product Studio bug from 2013. Pretty barren bug. Here's the relevant part:

> Bug 190575 - Windows Blue Bugs - ProfileDataObject test gives weird spew locally
I should also point out, it is very likely that this is just an issue with the debug object, and not the product.  The debug object is, I believe, only used in this test case.

Reply:
^ @boingoing should we open a new bug to track the relevant issue in a modern issue tracker and update the bug reference?

Reply:
If the tests still fail. If they're passing now let's just enable them?

Reply:
Tracking in https://github.com/Microsoft/ChakraCore/issues/4770 so this doesn't block this PR